man/systemd.service.xml

```diff
@@ -206,7 +206,7 @@
             identify the main process of the service. The manager will proceed with starting follow-up units
             after the parent process exits.</para></listitem>
 
-            <listitem><para>Behavior of <option>oneshot</option> is similar to <option>simple</option>;
+            <listitem><para>Behavior of <option>oneshot</option> is similar to <option>exec</option>;
```

Comment:
One major difference between `simple`, `exec`, and `oneshot` is that when the service is handled as started and when subsequent job is being started.
So, in that sense, both `oneshot` is similar to both `simple` and `exec`.
Maybe better to mention both types, rather than changing the type?

Reply:
> One major difference between simple, exec, and oneshot is that when the service is handled as started and when subsequent job is being started.

But isn't that anyway already "overridden" by `oneshot`’s:
> however, the service manager will consider the unit up after the main process exits

?
I thought that sentence basically overrides all of the `fork()`/`execvp()`/when-is-the-services-started-up stuff in `simple` and `exec`.

Reply:
Right. So, in that sense, there is nothing bad in the original sentence. And of course, your change is also correct.

Reply:
So… can this be resolved and merged?

Reply:
Not sure. I prefer to list both. Maybe better to wait for other reviewers?